torch/quantization/fx/quantization_patterns.py

```diff
@@ -725,41 +732,78 @@ def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
         if convert_custom_config_dict is None:
             convert_custom_config_dict = {}
         additional_static_quant_mapping = convert_custom_config_dict.get("static", {})
-        activation_post_process = quantizer.activation_post_process_map[node.name]
-        if node.op == 'call_module':
-            module = quantizer.modules[node.target]
-            module.activation_post_process = activation_post_process
-            quantized_module_cls = get_static_quant_module_class(
-                type(module), additional_static_quant_mapping)
-            quantized_module = quantized_module_cls.from_float(module)
-            parent_name, name = _parent_name(node.target)
-            setattr(quantizer.modules[parent_name], name, quantized_module)
-            return quantizer.quantized_graph.create_node(
-                'call_module',
-                node.target,
-                load_arg(quantized=[0])(node.args),
-                load_arg(quantized=False)(node.kwargs))
+
+        all_dtypes = [
+            (torch.quint8, torch.qint8, None),
+            (torch.float16, torch.float16, None)
+        ]
+        int8_dtypes = [
+            (torch.quint8, torch.qint8, None)
+        ]
+        supported_dtypes = {
```

Comment:
optional for a future PR, would it make sense for these mappings to live somewhere else in a mapping only file?

Reply:
yeah, I plan to refactor this part a bit later, maybe we can put the type tuple in a function. for mappings I think it would make sense to keep them here for now since they are related to the patterns above (in register_patterns), but maybe we can include them in the patterns, will need to think about it more. let me know if you have any suggestions.